### DISCUSSION TITLE
Thoughts on requiring a Neo-tree configuration with bug reports?

### DISCUSSION THREAD

**Discussion by miversen33** [nvim-neo-tree/neo-tree.nvim#1006] [Internal Development] [answered]
Its incredibly frustrating that we get so many bug reports with a screenshot and "X doesn't work".

Time is spent asking for a configuration and hoping the end user supplies one. It would probably be easier if we forced users to provide a neo-tree configuration with their bug reports. Hell, I have created a whole Neovim conf that can be used to create a whole new environment with the user's configuration so we can isolate and test their issue (or we can make the user try that before submitting their issue?)

Maybe I just haven't had enough coffee today but the amount of these kinds of issues is quite irritating.

**Comment by miversen33** [comment]
Probably should have tagged @cseickel @pysan3 @nhat-vo lol

**Comment by pysan3** [comment]
Totally agree with you :)

Maybe we could add issue templates, and here's some that came to my mind.

- folke's repos: https://github.com/folke/noice.nvim/tree/main/.github/ISSUE_TEMPLATE
- NvimTree: https://github.com/nvim-tree/nvim-tree.lua/tree/master/.github/ISSUE_TEMPLATE
  - This seems to be a nice template to copy, but I'd be happier if we use `lazy` for repro template.

**Reply by miversen33** [reply]
Do you have experience with this? I actually haven't played around much with Github outside of "I haz code, I haz issues" lol.

**Reply by pysan3** [reply]
Ok, I'll make a PR when I get back to my pc ;)

**Comment by cseickel** [comment, marked as answer]
100% agree, and the noice template is great. Go ahead and make one up for neo-tree.
